assets/javascripts/kitten/karl/pages/prelaunch/index.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Title, Separator, Text, Field, Button } from '../../../index'
+import { pxToRem } from '../../../helpers/utils/typography'
+
+const User = styled(({ className }) => (
+  <img className={className} src="https://picsum.photos/60" />
+))`
+  border-radius: 50%;
+  margin-top: ${pxToRem(-30)};
+  border: ${pxToRem(4)} solid #fff;
+`
+
+const Container = styled.div`
+  background-image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjg4MCIgaGVpZ2h0PSIxODAwIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIj48ZGVmcz48cGF0aCBpZD0iYSIgZD0iTTAgLjA4MWgyODgwdjE4MDBIMHoiLz48L2RlZnM+PGcgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj48cGF0aCBmaWxsPSIjMDAwIiBkPSJNMCAwaDI4ODB2MTgwMEgweiIvPjxnIG9wYWNpdHk9Ii42Ij48bWFzayBpZD0iYiIgZmlsbD0iI2ZmZiI+PHVzZSB4bGluazpocmVmPSIjYSIvPjwvbWFzaz48dXNlIGZpbGw9IiNDQUY0RkUiIGZpbGwtcnVsZT0ibm9uemVybyIgeGxpbms6aHJlZj0iI2EiLz48ZyBzdHJva2U9IiMyMjIiIHN0cm9rZS13aWR0aD0iMjMuNDQ4IiBtYXNrPSJ1cmwoI2IpIj48cGF0aCBmaWxsPSIjMTlCNEZBIiBkPSJNLTYwNS4yNjQtMzM2Ljg5Mkw4NDMuNzg5LTcyNS4xNjRsMzg4LjI3IDE0NDkuMDQzLTE0NDkuMDUzIDM4OC4yNzN6Ii8+PHBhdGggZmlsbD0icmVkIiBkPSJNLTIwMS41MzggMTEwOC4wMUwxMjMyLjA2IDcyMy44OGw3MTQuOTM1IDI2NjguMTcxLTE0MzMuNTk3IDM4NC4xMzF6Ii8+PHBhdGggZmlsbD0iI0ZGRUJFMCIgZD0iTTg1My42MjYtNjg4LjQ1NWwxNzc5LjQzOC00NzYuOEwzMTA5Ljg1OSA2MTQuMTdsLTE3NzkuNDM4IDQ3Ni44eiIvPjwvZz48L2c+PC9nPjwvc3ZnPg==);
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100vh;
```

Comment:
Au cas où le contenu est plus haut que la page, peut-être qu'il vaudrait mieux mettre 

```suggestion
  min-height: 100vh;
```